### ISSUE TITLE
Make the dashboard more role dependant

### ISSUE THREAD

**Issue by brainthee** [brainthee/CHAOTICA#6]
The dashboard should show only what's useful. I guess a bigger question is - based on the typical roles... what's useful?!

**Comment by tevers200** [comment]
Some thoughts on per-role dashboard requirements:
# Organisation Roles
## Manager
* View schedule overview for organisation (either "today" or "this week")
* Have an easy view of which consultants are not assigned to work today
* View any pending leave requests/scopes

## Service Delivery
Unsure - not quite sure what this role is used for

## Sales
* Overview of my jobs which are awaiting scoping
* Overview of my jobs which are currently in progress
* Overview of my upcoming jobs in the next week
* Overview of my upcoming jobs which have not yet been confirmed
* "Create Job" button

## Consultant
* Current Schedule overview
* Upcoming tasks view (e.g. pre-con checks)
* Current job link
* Overdue tasks (eg. job to TQA)

## TQA'er
* Jobs which are awaiting TQA (ordered by deadline)
* Jobs which are overdue for TQA
* Jobs which are awaiting Author Updates (ordered by deadline)
* Upcoming expected TQA jobs (this week)

## PQA'er
* Jobs which are awaiting PQA (ordered by deadline)
* Jobs which are overdue for TQA
* Jobs which are awaiting Author Updates (ordered by deadline)
* Upcoming expected PQA jobs (this week)

## Scoper
* Jobs awaiting scoping
* Jobs awaiting scope signoff

**Comment by tevers200** [comment]
As users can be assigned multiple roles, I guess the elements for all roles they're assigned should be shown, but this could make for quite messy dashboards.

**Comment by brainthee** [comment]
> As users can be assigned multiple roles, I guess the elements for all roles they're assigned should be shown, but this could make for quite messy dashboards.

True, but I'm hoping I can make it pretty jointed. E.g. certain tabs are only shown if you have the role. So the idea is the dashboard "layout" is the same for everyone - it's just the level of detail that changes

**Comment by tevers200** [comment]
> > ## Service Delivery
> > Unsure - not quite sure what this role is used for
> 
> Think of it like an operations team :)

Cool, updated my comment to cover the service delivery team then too :)
